.github/workflows/verify-fetchy.yml

```diff
@@ -0,0 +1,29 @@
+name: Verify Run Fetchy Fetch Boi
+on:
+  pull_request:
+
+jobs:
+  fetchy:
+    name: Verify Run Fetchy
+    runs-on: ubuntu-latest
+    env:
+      # set this secret, you can get the value from this discord message https://discord.com/channels/658523382442360866/903771050385502239/904216952141152328
+      DISCORD_WEBHOOK: $ {{ secrets.DISCORD_WEBHOOK }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2.3.4
+        with:
+          token: ${{ secrets.PAT }}
+      - name: Load global environment variables
+        run: cat .github/config/global.env >> $GITHUB_ENV
+      - name: Setup node
+        uses: actions/setup-node@v2.4.1
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: "npm"
+      - name: Install dependencies
+        run: npm ci
+      - name: Run Fetchy
+        run: npm run fetch-boi
+      - name: Print results
+        run: git ls-files --others --exclude-standard | xargs -I filename cat filename
```

Comment:
There's probably a better way ;)

Reply:
TBH this isn't getting printed out at the moment so we don't need this. 

But if you want you can just do `git status`. It should be enough to tell us a new file got generated.

Reply:
Interesting yeh ofc - we could purge the data folder prior to run, but like you say, probably not necessary

Reply:
```suggestion
        run: git status
```